app/lib/utils/utils.js

```diff
@@ -1,14 +1,35 @@
-function areEqual(queryItem, item) {
-  return (queryItem.localeCompare(item, 'en', { sensitivity: 'base' }) === 0);
-}
+const constants = require('../constants');
 
 function getValues(myHash) {
   const keys = Object.keys(myHash);
 
   return keys.map(v => myHash[v]);
 }
 
+function getAlternativeTypeFor(type) {
+  if (type === constants.serviceTypes.professional) {
+    return constants.serviceTypes.kit;
+  } else if (type === constants.serviceTypes.kit) {
+    return constants.serviceTypes.professional;
+  }
+  return type;
+}
+
+function areEqual(queryItem, item) {
+  return (queryItem.localeCompare(item, 'en', { sensitivity: 'base' }) === 0);
+}
+
+function isProfessionalChoice(query) {
+  return (areEqual(query.type, constants.serviceTypes.professional)
+    && (getValues(constants.serviceChoices).includes(query.origin)));
+}
+
+function getResultsInternalLink(query, location) {
```

Comment:
With the change from `link` to `url` for both `resultsInternal___` and `resultsExternal___`, can this be updated to `getResultsInternalUrl` for consistency please?